### PR TITLE
Update to Members Guide

### PR DESCRIPTION
**Pull request by Aeisharat**
This update is to bring the member's guide in line with current policy at the Hackspace, and to expedite making it available as part of the new member joining process as soon as possible.

Also I just rewrote a massive chunk of the sodding thing.

### CODE REVIEW

File: children.rst [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+Children are welcome at Nottinghack, however be aware that it is primarily an adult environment. While it is unlikely that there would be content overtly unsuitable for children, members may not be watching their language or could be discussing adult topics. In addition to this; the Hackspace has potentially dangerous tools and chemicals which are not locked away, and no specific child-proofing measures in place.
 
-Young adults aged between 16 and 18 can join, but one of their parents or guardians will need to meet with a trustee at the Hackspace to understand the risks involved.
+**Under-18s must be accompanied by an adult**, and are generally expected to be engaged with the space some way (such as working on a project). Periodically we have events specifically for children of all ages.
 
-Periodically, we do have events specifically for children of all ages.
\ No newline at end of file
+.. __: membership@nottinghack.org.uk
```
Comment (geeksareforlife) [inline children.rst]: should be

mailto:membership@nottinghack.org.uk

File: children.rst [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+Children are welcome at Nottinghack, however be aware that it is primarily an adult environment. While it is unlikely that there would be content overtly unsuitable for children, members may not be watching their language or could be discussing adult topics. In addition to this; the Hackspace has potentially dangerous tools and chemicals which are not locked away, and no specific child-proofing measures in place.
 
-Young adults aged between 16 and 18 can join, but one of their parents or guardians will need to meet with a trustee at the Hackspace to understand the risks involved.
+**Under-18s must be accompanied by an adult**, and are generally expected to be engaged with the space some way (such as working on a project). Periodically we have events specifically for children of all ages.
```
Comment (geeksareforlife) [inline children.rst]: Change to:

**Under-16s, and 16-18 year olds who are not 'Young Hackers', must be accompanied by an adult**

File: comfy.rst [hunk 3]
```diff
@@ -1,6 +1,8 @@
 Comfy Area
 ==========
 
-The first room you enter at Nottingham Hackspace is the comfy area, where people tend to congregate and socialise.
+The comfy are is a place where people tend to congregate and socialise. With a project, soft furnishings, coffee tables, and shelves full of books, it's an excellent place to relax.
```
Comment (geeksareforlife) [inline comfy.rst]: projector

File: cleaning-improving.rst [hunk 4]
```diff
@@ -1,22 +1,12 @@
 Cleaning and Improving the Space
 ================================
 
-The hackspace is run, cleaned, organised, maintained by YOU - thats every hackspace member.
-
-One of the best things to do as a member is to clear-up just a little bit more, that way the space is more usable for everyone.
+**The Hackspace is run, cleaned, organised, and maintained by every Hackspace member.** One of the best things to do as a member is to clear-up just a little bit more mess than you may make, that way the space is more usable for everyone and continues to look good.
 
 Work-surfaces that are regularly used, such as the benches in the workshop & the large table in the blue room, are designated as clear zones. Please make sure to clear these areas after working on them.
```
Comment (geeksareforlife) [inline cleaning-improving.rst:7]: Why have you removed clean the space tuesdays?

File: health-and-safety.rst [hunk 5]
```diff
@@ -1,20 +1,15 @@
 Health & Safety
 ===============
 
-Health & Safety is the responsibility of every member of Nottingham Hackspace.
+**Health & Safety is the responsibility of every member of Nottingham Hackspace.** It is the duty of each member to make sure they are safe and not endangering the people around them.
 
-The hackspace has a lot of potentially dangerous tools available to members, and rather than mess around with them, you should ask to be inducted on any machine or tool you haven’t used before. Generally, another member will be available and willing to do this; the best places to ask are the mailing list or at one of the open hack nights.
+The Hackspace has a lot of potentially dangerous tools available to members; if you are not confident with a tool or machine, or haven't used it before, you should ask to be inducted on it. For machines without specific inductions (see the Teams section), another member will typically be willing to provide advice; the best places to ask are the mailing list or an Open Hack event.
```
Comment (geeksareforlife) [inline health-and-safety.rst]: We call it "Google Group" elsewhere

File: online.rst [hunk 6]
```diff
@@ -1,70 +1,66 @@
...
 
+Nottingham Hackspace maintains a wide online presence, with sites ranging from our social media and mailing list to our wiki and internal tools like HMS. Our online presence is one of the main ways we attract new members, communicate with existing ones, and promote our events and workshops. If you're able to help promote us from time to time whether by retweet, share or like, it is a big help toward attracting new people.
+
+We welcome contributions to all these sites; if you'd like to get involved with any please contact the relevant team, who's details can be found in Section 1.3 - Hackspace Organisation.
+
 Websites
 --------
 
-Main website
+Main website - www.nottinghack.org.uk
```
Comment (geeksareforlife) [inline online.rst:11]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 7]
```diff
@@ -1,70 +1,66 @@
...
+This is our main public-facing site; where events, workshops and links to other online resources are available. 
 
-Wiki
-````
-We run a wiki here: https://wiki.nottinghack.org.uk/wiki/NottingHack.  You can use your HMS account details to log into the wiki and make edits.
+The website is managed by our Comms team.
 
-As a new member you should have received a link in your email telling you how to sign up for a Wiki account and submit a feed to our planet - used for blog entries.
+Hackspace Management System (HMS) - lspace.nottinghack.org.uk/hms/
+`````````````````````````````````
```
Comment (geeksareforlife) [inline online.rst]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 8]
```diff
@@ -1,70 +1,66 @@
...
-If you blog about hackspace relevant stuff, you’d be most welcome to contribute your feed to Planet Nottinghack. Many members run their own blogs, & this automatically uses RSS to collect them all into one place: http://planet.nottinghack.org.uk/.  If you would like to be included on this, you just need to open a pull request against the GitHub repository.  If you’re not sure how to do this then ask on the google group.
+To log in, please click the link titled: “Already a member? Login”, then enter your username and password.
 
-Hackspace Management System (HMS)
-`````````````````````````````````
-HMS was developed by hackspace members to organise and manage our ever-increasing group. Every hackspace member has an account on HMS which is accessible here:
+HMS is managed by the Software team.
 
-https://lspace.nottinghack.org.uk/hms/
+Wiki - wiki.nottinghack.org.uk/wiki/NottingHack
```
Comment (geeksareforlife) [inline online.rst:25]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 9]
```diff
@@ -1,70 +1,66 @@
...
+````
+The Nottinghack Wiki is where we store information around projects and equipment, as well as organisation details for big events such as Makerfaire and EMF. As a member please feel free to start a user page to hold build logs of your projects, or when you're collaborating with other members on projects and need a place to store information.
 
-Then click “Already a member? Login”.
+You can use your HMS account details to log into the wiki and make edits.
 
-Within HMS you can check your Snackspace account (more about that later) and book various tools.
+Planet Nottinghack - planet.nottinghack.org.uk/
+``````````````````
```
Comment (geeksareforlife) [inline online.rst]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 10]
```diff
@@ -1,70 +1,66 @@
...
+If you blog about Hackspace relevant stuff, projects you've made here or the subject of making and hacking as a whole, you’d be most welcome to contribute your feed to Planet Nottinghack. Many members run their own blogs, and Planet Nottinghack uses RSS to collect them all into one place.  If you would like to be included on this, please open a pull request against the GitHub repository.  If you’re not sure how to do this then ask on the Google Group.
 
-Members Gudie
+Members Guide - guide.nottinghack.org.uk
```
Comment (geeksareforlife) [inline online.rst:35]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 11]
```diff
@@ -1,70 +1,66 @@
...
+This members guide is available online, where you can also download PDF and Epub versions. 
 
-From there you can also download PDF and Epub version
+Want to help contribute to this guide? Make a pull request against it on GitHub or talk to the Membership team.
 
 Other forms of interaction
 --------------------------
 
-Google Mailing List
+Google Group - groups.google.com/group/nottinghack
```
Comment (geeksareforlife) [inline online.rst:44]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 12]
```diff
@@ -1,70 +1,66 @@
...
 ```````````````````
-The main place we have discussions is the public google group, which is open to anyone: https://groups.google.com/group/nottinghack
-
-Facebook
-````````
-We also have a facebook page here: https://www.facebook.com/nottinghack
+The main place for discussion outside of the Hackspace itself is our Google Group mailing list, which is open to contributions from anyone. Please be aware that the mailing list is 
 
-Hackspace Newsletter
+Hackspace Newsletter - newsletter.nottinghack.org.uk
```
Comment (geeksareforlife) [inline online.rst:48]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 13]
```diff
@@ -1,70 +1,66 @@
...
 ````````````````````
 There is a monthly newsletter which is a great place to hear about upcoming events and hackspace related activity. You can sign up to this newsletter here:
 
 http://nottinghack.us5.list-manage1.com/subscribe?u=a4e59e4c29bd40e76419a037b&id=0a6da449c9
 
-Twitter
+Facebook - www.facebook.com/nottinghack
+````````
```
Comment (geeksareforlife) [inline online.rst]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 14]
```diff
@@ -1,70 +1,66 @@
...
+Our Facebook page is one of the main ways we promote ourselves to potential new members, whether by sharing photos and videos of events or telling the world about our next big project.
+
+Twitter - @HSNotts
```
Comment (geeksareforlife) [inline online.rst:58]: Need to extend the backticks to cover the entire title

File: online.rst [hunk 15]
```diff
@@ -1,70 +1,66 @@
...
 ```````
-There are two Twitter accounts relevant to the hackspace:
-* @HSNotts is the general Twitter account,
-* @Holly533MHz is the voice of our server, tweeting about the various sensors built into the space. The latter account is locked down and we need to verify you as a member before it’ll accept a follow request.
+Our Twitter account works in a similar way to our Facebook page, only with 140 characters or less. We'll often tweet while at external events, with pictures of members representing Nottinghack or projects we've taken to demonstrate.
 
-Flickr
+Flickr - www.flickr.com/groups/nottinghack/
```
Comment (geeksareforlife) [inline online.rst:62]: Need to extend the backticks to cover the entire title

File: organisation.rst [hunk 16]
```diff
@@ -1,53 +1,111 @@
...
-The day-to-day running of the hackspace is done by you and every member.
-The various activities are run by teams, which are outlined here:
+The day-to-day running of the Hackspace is done by every one of our members. Even quick and simple things like tidying up or putting tools back into their proper storage are a great help.
+
+More specific activites are managed by teams within the Hackspace, a list of which can be found below. Please remember the Hackspace is a community and not a service; we can't guarantee tool uptime or material availability. The teams are all members like yourself, who volunteer their time to help make the space better.
+
+Want to get involved in a team? Drop them an email, or ask them when they're in the space.
 
-Safety
+Safety - safety@nottinghack.org.uk
```
Comment (geeksareforlife) [inline organisation.rst:20]: Need to extend the backticks to cover the entire title

I'm going to stop adding this comment - it needs changing on all titles on this page

File: children.rst (hunk 1 above)
Comment (Aeisharat) [inline children.rst]: Added

File: online.rst (hunk 8 above)
Comment (Aeisharat) [inline online.rst:25]: Fixed this one, but git hasn't noticed for some reason.

**Comment by geeksareforlife** [comment]
Looking good with the updates

Going to merge this and any other little issues can be picked up seperately

**Pull request merged**